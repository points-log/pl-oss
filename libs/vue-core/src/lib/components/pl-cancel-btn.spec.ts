import { shallowMount } from '@vue/test-utils';
import Vue from 'vue';
import Vuetify from 'vuetify';
import PLCancelBtn from './pl-cancel-btn.vue';

Vue.use(Vuetify);

describe('PLCancelBtn', () => {
  it('should match the snapshot', () => {
    const wrapper = shallowMount(PLCancelBtn);
    expect(wrapper.element).toMatchSnapshot();
  });
});
