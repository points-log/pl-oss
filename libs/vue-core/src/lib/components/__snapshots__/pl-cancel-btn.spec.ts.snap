// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PLCancelBtn should match the snapshot 1`] = `
<v-btn-stub
  activeclass=""
  tag="button"
  text="true"
  type="button"
>
  Cancel
</v-btn-stub>
`;
